docs/getting_started/troubleshooting.md

```diff
@@ -26,14 +26,23 @@ For instance, if one of those Fields is configured to accept files up to 10MB, t
 
 You also need to define settings for uploading files in `php.ini`: `upload_max_filesize` and `post_max_size`.
 
-## Cloning failed using an ssh key
+## Composer packages: Cloning failed using an ssh key
 
 When dealing with [updates.ez.no](http://updates.ez.no) packages, you may get a "Cloning failed using an ssh key" error
 if you tell Composer to download dev packages or to download from source.
 [updates.ez.no](http://updates.ez.no) currently supports only distribution packages in alpha stability or higher.
 
 To avoid the error, check the stability of packages and avoid using `--prefer-source`.
 
+## Redis: Cache / Session data inconsistant across web servers
+
+See [Redis Cluster info in persistence cache doc](../guide/persistence_cache.md), and make sure you only read/write to active master instance.
+
+## Redis: Session are removed or new sessions are refused
+
+See info on [Redis in session doc](../guide/session.md); Ideally use an unique instance of Redis for sessions, that either never runs of of memory or uses an eviction policy that suites your needs.
```

Comment:
`separated instance` perhaps? Sounds more natural for me. 

Nitpics: 
`an unique` -> `a unique`
`runs of of memory` -> `run out of memory`
`suites` -> `suits`